core/src/main/kotlin/services/OrchestratorService.kt

```diff
@@ -53,17 +53,17 @@ class OrchestratorService(
         path: String?,
         jobConfig: JobConfigurations,
         jobConfigContext: String?,
-        labels: Map<String, String>?
+        labels: Map<String, String>?,
+        jwt: String?
     ): OrtRun {
         val ortRun = db.dbQuery {
             ortRunRepository.create(repositoryId, revision, path, jobConfig, jobConfigContext, labels.orEmpty())
         }
 
-        // TODO: Set the correct token.
         orchestratorSender.send(
             Message(
                 header = MessageHeader(
-                    token = "",
+                    token = jwt.orEmpty(),
```

Comment:
Hmm, if empty tokens are not supported, we probably should not fall back to one here, or?

Reply:
They are supported in all transports except SQS.